package.json

```diff
@@ -36,7 +36,7 @@
     "esbuild": "^0.23.0",
     "fs-extra": "^11.2.0",
     "gulp-sort": "^2.0.0",
-    "i18next": "^23.5.1",
+    "i18next": "^23.5.1 || ^24.0.0",
```

Comment:
Would it make sense to stop supporting v23 entirely instead?
If so, since that would be a breaking change, would it make sense to also move `i18next` to a peerDependency? It would help upstream projects with their dependency management.

Reply:
> Would it make sense to stop supporting v23 entirely instead?

Your call. I seemed easy enough to support both, and it would make my life easier as I have various packages using i18next that depend on this, so I could update things gradually. But I'd be happy to change things if wanted.

> If so, since that would be a breaking change, would it make sense to also move `i18next` to a peerDependency? It would help upstream projects with their dependency management.

I don't quite understand how peerDependency could help as I've never worked with that. Note that i18next-parser is importing/using i18next directly and is not plugin as noted here: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependencies but maybe I'm missing something.